orbs/helm-runner/orb.yml

```diff
@@ -145,7 +145,14 @@ jobs:
         default: "./k8s/$CIRCLE_PROJECT_REPONAME"
     executor: "deploy"
     steps:
-      - checkout
+      - run:
+          name: "Git checkout"
+          command: |
+            git init
+            git sparse-checkout init --cone
+            git sparse-checkout set k8s/
+            git remote add origin $CIRCLE_REPOSITORY_URL
+            git pull --depth=1 origin $CIRCLE_BRANCH
```

Comment:
Very interesting ! I am wondering if it could not be worth making this a command in `github` orb so that we can reuse it in other projects where a complete checkout is not necessary. We could define some parameter such as the list of folders to include in the checkout...

WDYT?

Reply:
Yes definitely, I'll look into it :)